codec-fast/README.md

```diff
@@ -0,0 +1,20 @@
+# FAST codec
+
+This codec can be used for decoding and encoding message in FAST protocol
+
+## Configuration
+
+### Codec factory
+
+To use the following codec you need to specify the following codec factory:
```

Comment:
To use the following codec **,** you **will** need to specify the following codec factory: